sdk/python/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb

```diff
@@ -858,9 +1003,9 @@
    "name": "python3-azureml"
   },
   "kernelspec": {
```

Comment:
nit: we should probably not modify this, since gates running notebook validation make use of kernel spec to create env on which notebooks are run. Please confirm with Jeff once on this, if you are planning to modify. Thank

Reply:
Resolved